src/utils/getOrganizations.ts

```diff
@@ -0,0 +1,15 @@
+import { ScaffoldedContext } from './next';
+import { ZetkinMembership } from 'types/zetkin';
+
+const getOrganizations = async (ctx: ScaffoldedContext): Promise<number[]> => {
```

Comment:
I think this function's name does not really reflect what it does. I think something like `getUserOrganizations` or `getUserMemberships` would be more suitable. Do you mind renaming it?

Reply:
Would love to rename 😸